### DISCUSSION TITLE
Is this plugin support tailwind?

### DISCUSSION THREAD

**Discussion by yushaku** [uga-rosa/ccc.nvim#94] [Q&A]
how can i config to enable show color when using tailiwind?

**Comment by thenbe** [comment]
A similar plugin ([mini.hipatterns](https://github.com/echasnovski/mini.hipatterns)) can be used to highlight tailwind classes. However, it doesn't have a color picker.

Click ~["Full Spec"](https://www.lazyvim.org/extras/util/mini-hipatterns)~ [here](https://github.com/LazyVim/LazyVim/blob/68ff818a5bb7549f90b05e412b76fe448f605ffb/lua/lazyvim/plugins/extras/util/mini-hipatterns.lua) to see source code for how to highlight tailwind classes.

EDIT: updated link

**Comment by uga-rosa** [comment]
First of all, I must warn you that I have never written tailwind, so I may not know the syntax of the colors you intend to use.
This plugin itself does not perform complex parsing; it only works with colors defined in CSS Color Module level4, such as #ff00ff, for which the color can be determined by that string alone (but there are also pickers such as custom_entries. Please read the documentation for more details).
However, since nvim-lsp is supported, more complex colors can be highlighted and picked if you are using a language server that will parse and pass color information.
